src/Controller/WopiController.php

```diff
@@ -42,13 +41,21 @@ function wopiCheckFileInfo(string $id, Request $request) {
             return static::permissionDenied();
         }
 
+        /** @var \Drupal\media\MediaInterface|null $media */
+        $media = \Drupal::entityTypeManager()->getStorage('media')->load($id);
+        if (!$media) {
+            // @todo Use default mechanism for access denied response.
+            return static::permissionDenied();
```

Comment:
Comment should be changed.

Reply:
I think this was referencing a `@todo` comment, which was removed since.
So this comment was done.